### PR TITLE
[TASK] Use category for site set setting

### PR DESCRIPTION
**Pull request by ErHaWeb**
In TYPO3 v13.4 I would now recommend giving the settings a category, otherwise they will be displayed under “Other”. It also makes sense to use a labels.xlf with a defined key format.

### CODE REVIEW

**Comment by brotkrueml** [comment]
Thanks, will have a look in the next days :+1:

**Comment by ErHaWeb** [comment]
It might also make sense to demonstrate sub-categories, but that might go too far beyond the original topic of the blog post. I had made two of my v12/13 compatible extensions v13 ready based on your blog post and recently made the extension noted here.

See:
https://github.com/ErHaWeb/feed_display/commit/4a494ae0a906806b72386af1d5d030026cdb0958
https://github.com/ErHaWeb/klaro_consent_manager/commit/deeb3a24af58ecb4006460e398563470588557d1

**Comment by brotkrueml** [comment]
Hmm, I would like to avoid the additional complexity with the translated label - and to provide a first successful experience. I would rather add a link to the documentation in the blog article once it has catched up with the translated labels. The category is fine for me.

What do you think?

**Comment by ErHaWeb** [comment]
I can follow and understand your reasoning. My concern was to provide the completely “correct” status. However, I did not take into account the additional complexity when entering the topic. In the end, we should at least do the categorization to get the settings out of the “Other” block.

File: Configuration/Sets/SiteSetsCompatExample/settings.definitions.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
+categories:
+  sitesetscompat:
+    label: 'Site sets compat example'
```
Comment (brotkrueml) [inline Configuration/Sets/SiteSetsCompatExample/settings.definitions.yaml:3]:
```suggestion
    label: "Site sets compat example"
```

**Comment by brotkrueml** [comment]
@ErHaWeb Thanks. Without the translations it may be easier to understand the transition away from constants, instead using some magic.

**Pull request merged**